### DISCUSSION TITLE
Link Event Handlers

### DISCUSSION THREAD

**Discussion by schlopmyflop** [cosmograph-org/cosmos#85] [General]
Hi,

I've been exploring various graph visualization libraries over the past few days and have been loving this project! My only issue is the lack of event handlers tied to links (hover, click). Are there any plans to add the same functionality to links?

**Comment by rokotyan** [comment]
Thanks @schlopmyflop! Adding hover and lick functionality is not on our roadmap right now, but I agree that it's an absolute must for a graph visualization library. We'll see if it's something we can add.
